src/Lock.php

```diff
@@ -79,23 +80,31 @@ public static function showForItem(CommonDBTM $item)
         global $CFG_GLPI, $DB;
 
         $ID       = $item->getID();
-        $itemtype = $item->getType();
-        $header   = false;
+        $itemtype = $item::class;
 
         //If user doesn't have update right on the item, lock form must not be displayed
         if (!$item->isDynamic() || !$item->can($item->fields['id'], UPDATE)) {
             return false;
         }
 
-        echo "<div class='alert alert-primary d-flex align-items-center' role='alert'>";
-        echo "<i class='fas fa-info-circle fa-xl'></i>";
-        echo "<span class='ms-2'>";
-        echo __("A locked field is a manually modified field.");
-        echo "<br>";
-        echo __("The automatic inventory will no longer modify this field, unless you unlock it.");
-        echo "</span>";
-        echo "</div>";
+        // language=Twig
+        $list_info_alert_template = <<<TWIG
+            <div class="alert alert-primary d-flex align-items-center" role="alert">
```

Comment:
```suggestion
            <div class="alert alert-info d-flex align-items-center" role="alert">
```

Not a recent issue, but the style primary for alert is unreadable for light theme (especially with auror theme)